Jenkinsfile

```diff
@@ -432,7 +432,8 @@ ansiColor('xterm') {
                   version = version.trim()
                   echo "next version is ${version}"
                   sh 'npm run build'
-                  sh 'npm run build:script'
```

Comment:
Changing from build to build:script makes me nervous

Reply:
build still builds the packages
build:script is a different and separate process